install/third-party/confluent-cloud/install.yml

```diff
@@ -0,0 +1,13 @@
+id: third-party-confluent-cloud
+name: ConfluentCloud
+title: Confluent Cloud Managed Kafka
+description: |
+  Observability for Confluent Cloud's fully managed service for Apache Kafka on New Relic delivered using the OpenTelemetry Collector
+target:
+  type: integration
+  destination: cloud
+
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud
```

Comment:
Looks like this link doesn't exist yet - can we update this to a working URL?

Reply:
@josephgregoryii We're just waiting on the docs team to publish the updates and then the URL will be live. I'll post here when it is done.

Reply:
@josephgregoryii The docs are live! https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud/